### ISSUE TITLE
BMS: Generic connection info field

### ISSUE THREAD

**Issue by ReinvdZee** [victronenergy/gui-v2#1158]
Add the connection information field, if valid. 

See this issue: 
https://github.com/victronenergy/venus/issues/1252
And this commit: 
https://github.com/victronenergy/gui/commit/cc83826becd84da986237a2e18a4cb3b662cf048

This is a recent feature, gui-v1 part not merged into venus yet.

**Comment by DanielMcInnes** [comment]
The changes for this are in branch: 1158-bms-generic-connection-info-field. Not tested yet.

**Comment by mpvader** [comment]
we removed this from v1.0.0, as it should only be progresses further if it can be tested easily and is not urgent for us.

**Comment by ReinvdZee** [comment]
Gui-v1 & backend part are merged per Venus OS V3.40~30. Unblocking
